src/library/trackset/baseplaylistfeature.cpp

```diff
@@ -600,6 +600,101 @@ void BasePlaylistFeature::slotExportPlaylist() {
     }
 }
 
+void BasePlaylistFeature::slotExportPlaylists() {
+    const PlaylistDAO::HiddenType hidden = PlaylistDAO::PLHT_NOT_HIDDEN;
+
+    QList<QPair<int, QString>> allPlaylists = m_playlistDao.getPlaylists(hidden);
+
+    QPair<int, QString> thePair;
```

Comment:
Such an anonymous QPair is always a pain in reviews. I would be happy if you would go the extra mile to create a named struct for it.

Reply:
Can you explain a bit on what you mean by an 'anonymous QPair'?

Reply:
> Such an anonymous QPair is always a pain in reviews. I would be happy if you would go the extra mile to create a named struct for it.

Still confused on this

Reply:
A QPair can contain everything. 
It's members "first' and "second' are badly named, in every context. 

The proposed code lints that by hiding this ugly detail of QPair. 

The alternative to QPair is a custom struct, with properly named members. This should be used whenever the Pair escapes the scope of just being a return value.